Microsoft.DotNet.Interactive.PowerShell/Host/PSKernelHostUserInterfaceProgress.cs

```diff
@@ -95,7 +102,7 @@ internal void ResetProgress()
 
         private void ProgressPaneUpdateTimerElapsed(object sender)
         {
-            Interlocked.CompareExchange(ref progPaneUpdateFlag, 1, 0);
+            Interlocked.CompareExchange(ref progPaneUpdateFlag, ToRender, ToNotRender);
```

Comment:
Should this just be `Interlocked.Exchange` since we don't actually use the result?

Reply:
Yeah, it can be just `Exchange`, but I guess it doesn't really matter :)